helpers/spark-shell.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import pwd
+
+import utils
+
+USER_HOME_DIR_ENT_IDX = 5
+
+if __name__ == "__main__":
+    logging.basicConfig(
+        format="%(asctime)s %(levelname)s %(message)s", level=logging.INFO
+    )
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--username", default=None, type=str, help="Username of service account to use."
```

Comment:
NOTE: Regarding parity with spark-submit, following optional parameters have not been added.

--deploy-mode - Discussed with Enrico. Not appropriate here.
--log-level - Will piggyback with other review comments changes